Datastore/tests/System/DatastoreSessionHandlerTest.php

```diff
@@ -52,6 +52,7 @@ public function testSessionHandler()
 
         $res = $client->runQuery($q, [
             'namespaceId' => $namespace,
+            'databaseId' => '',
```

Comment:
Changing `databaseId` to a non-empty string causes a failure, checking with the team how to get past this issue.

```
google-cloud-php/Datastore % XDEBUG_MODE=debug vendor/bin/phpunit -c phpunit-system.xml.dist                                           PHPUnit 8.5.29 #StandWithUkraine

......E..............................................             53 / 53 (100%)

Time: 22.76 seconds, Memory: 14.00 MB

There was 1 error:

1) Google\Cloud\Datastore\Tests\System\DatastoreSessionHandlerTest::testSessionHandler
Google\Cloud\Core\Exception\NotFoundException: {
  "error": {
    "code": 404,
    "message": "The database database-gcloud_testing_6319c7a74363c does not exist for project chatbots-344719 Please visit https://console.cloud.google.com/datastore/setup?project=database-gcloud_testing_6319c7a74363c to add a Cloud Datastore or Cloud Firestore database. ",
    "status": "NOT_FOUND"
  }
}


/Users/vishwarajanand/github/google-cloud-php/Datastore/vendor/google/cloud-core/src/RequestWrapper.php:368
/Users/vishwarajanand/github/google-cloud-php/Datastore/vendor/google/cloud-core/src/RequestWrapper.php:207
/Users/vishwarajanand/github/google-cloud-php/Datastore/vendor/google/cloud-core/src/RestTrait.php:97
/Users/vishwarajanand/github/google-cloud-php/Datastore/src/Connection/Rest.php:111
/Users/vishwarajanand/github/google-cloud-php/Datastore/src/Operation.php:483
/Users/vishwarajanand/github/google-cloud-php/Datastore/vendor/google/cloud-core/src/Iterator/PageIteratorTrait.php:238
/Users/vishwarajanand/github/google-cloud-php/Datastore/src/EntityPageIterator.php:58
/Users/vishwarajanand/github/google-cloud-php/Datastore/vendor/google/cloud-core/src/Iterator/ItemIteratorTrait.php:136
/Users/vishwarajanand/github/google-cloud-php/Datastore/tests/System/DatastoreSessionHandlerTest.php:61

ERRORS!
Tests: 53, Assertions: 146, Errors: 1.
google-cloud-php/Datastore %
```

Reply:
Seeing Internal error with GRPC client. Product team is working on it.

Reply:
Also, I've added a new test class `DatastoreSessionHandlerMultipleDBTest` which covers this scenario.